data/modules/data/general.yml

```diff
@@ -9,7 +9,7 @@ checkLists:
     - question: We have considered how to provide sensible access control to customers, database users and systems, [via the principle of least privilege](https://searchsecurity.techtarget.com/definition/principle-of-least-privilege-POLP).
     - question: We have setup [audit logs](https://logz.io/blog/audit-logs-security-compliance/) and are forwarding them via our central log forwarder.
     - question: We have read the [Data Classification Policy]() to determine if our data is classified as sensitive.
-  Sensitive_Data:
+  Sensitive Data:
```

Comment:
TIL spaces in YAML keys are ok :o

Reply:
TIL too. I suppose it makes sense since it's allowed in JSON

Reply:
Yeah 🤔